ft-sdk/src/utils.rs

```diff
@@ -0,0 +1,31 @@
+//! Utility functions for the SDK
+
+/// debug print queries generated by diesel for the postgres backend
+///
+/// # Example
+///
+/// ```rust
+/// use diesel::prelude::*;
+///
+/// diesel::table! {
+///     use diesel::sql_types::*;
+///
+///     user (id) {
+///         id -> Int8,
+///         name -> Text,
+///     }
+/// }
+///
+/// let conn = ft_sdk::default_pg().unwrap();
+///
+/// let query = user::table
+///     .select(user::name)
+///     .filter(user::id.eq(9))
+///     .limit(1);
+///
+/// ft_sdk::utils::dbg_query(&query);
+/// ```
+pub fn dbg_query<T: diesel::query_builder::QueryFragment<diesel::pg::Pg>>(query: &T) {
+    #[cfg(feature = "debug")]
+    ft_sdk::println!("{:?}", diesel::debug_query::<diesel::pg::Pg, _>(query));
+}
```

Comment:
We should also add a `ft_sdk::dbg!()`, which acts like acts like `dbg!()`. Actually, can we not make `println!()`, `dbg!()`, `log!()` etc use our internal helper, so we do not have to ask users to update their `println!()` calls to `ft_sdk::println!()` calls?